landoapi/landing_worker.py

```diff
@@ -171,9 +172,20 @@ def start(self):
                 db.session.commit()
 
                 repo = repo_clone_subsystem.repos[job.repository_name]
+
+                # Set up config override
+                config_override = {}
+                if repo.config_override:
+                    config_override.update(repo.config_override)
+
+                # Set `hg fix` config if `formatting` kwarg is set
+                if repo.formatting:
+                    for k, v in repo.formatting.items():
+                        config_override[f"fix.{k}"] = v
```

Comment:
This is a little ugly and I'm sure there's a better way.

Reply:
I think it would be a bit cleaner to merge this config within `HgRepo.__init__` rather than doing it here.